src/model.py

```diff
@@ -0,0 +1,92 @@
+
+import argparse
+import logging.config
+import os
+import sys
+
+import sqlalchemy
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy import Column, Integer, String, MetaData
+
+sys.path.append(os.path.abspath(os.path.join('..')))
+
+from config import SQLALCHEMY_DATABASE_URI, DATABASE_NAME
+
+
+logger = logging.getLogger(__name__)
+logger.setLevel("INFO")
```

Comment:
Setting logger levels should not be a part of the module scripts and should only be controlled by the master script or the config file.